themes/bootstrap5/templates/combined/results-list.phtml

```diff
@@ -45,19 +45,24 @@
 
   // Update result counts in jump links
```

Comment:
This comment should be a bit broader now; not just counts being updated.

Reply:
Good point! Let me know what you think about the new version...

Reply:
Looks good!